mlbench_core/api.py

```diff
@@ -409,9 +394,11 @@ def create_run(
         num_cpus=2.0,
         max_bandwidth=1000,
         image=None,
+        backend=None,
         custom_image_name=None,
         custom_image_command=None,
-        custom_image_all_nodes=False,
+        custom_backend=None,
+        run_all_nodes=False,
```

Comment:
I think the default should be `None` and having both this and a non custom backend specified should raise an exception that they're mutually exclusive.

Reply:
Do you mean for `run_all_nodes` ? or `backend`